java-client/src/main/java/org/opensearch/client/json/JsonpUtils.java

```diff
@@ -81,15 +81,30 @@ public <T> void serialize(T value, JsonGenerator generator) {
                 return;
             }
 
+            if (value instanceof String) {
```

Comment:
@playLeo I don't really follow this change: the `JsonpUtils::serialize(T value, JsonGenerator generator)` is called by OpenSearch Java model classes, which have to implement `JsonpSerializable`. It is never called on arbitrary types (like String, ...), since the model classes use `JsonGenerator` for those instead.

Reply:
Oh, I see, this is `JsonData` class, which does serialize raw values, I think the change should be contained with `JsonDataImpl`  (as it does with JsonValue now).